breeze.json

```diff
@@ -62,7 +62,7 @@
         ]
     },
     "org.kde.kdecoration2": {
-        "blur": false,
+        "blur": true,
```

Comment:
The problem is fixed.

But, for konsole, can this value depend on the `$HOME/.local/share/konsole/[using colorscheme] > [General] > Blur` value ?

Reply:
I don't think so. Transparency without blur looks awful anyways.

Reply:
Ok, this PR seems good enough now.
Thank you for your quick reply !